src/providers/LastSearchTermsProvider.js

```diff
@@ -6,26 +6,11 @@ import { useNamespace } from '@folio/stripes/core';
 import { LastSearchTermsContext } from '../contexts';
 import { getItem, setItem } from '../storage';
 import { INIT_PAGE_CONFIG } from '../hooks/useInventoryBrowse';
-import { useLogout } from '../hooks';
 import { segments } from '../constants';
 
 const LastSearchTermsProvider = ({ children }) => {
   const [namespace] = useNamespace();
 
-  const resetStorageAfterLogout = () => {
```

Comment:
Why is this function removed if we need to clear search term and results when logging out/switching affiliation?

Reply:
This function was added to avoid having search terms after a user logs out and then logs back in ([Scenario 1.1](https://issues.folio.org/browse/UIIN-2359)). By listening to the `LOGIN` event we get the same result.